sda/cmd/download/swagger_v1.yml

```
@@ -0,0 +1,265 @@
+openapi: 3.0.3
+info:
+  title: SDA file download API
+  version: "1.0"
+paths:
+  /file:
+    get:
+      description: Returns the requested file to the user
+      parameters:
+        - in: header
+          description: "Public key used to re-encrypt the file (header) with. This should be supplied as the base64 encoding of the PEM (RFC 7468) encoding of the public key"
```

The description `Public key used to re-encrypt the file (header) with.` is ambiguous, unclear whether the public key is used to encrypt just the file header or the entire file.

 ```suggestion
          description: "Public key used to re-encrypt the file header server-side before transmission. This should be supplied as the base64 encoding of the PEM (RFC 7468) encoding of the public key"
```